src/arrayMethodJoin.js

```diff
@@ -6,6 +6,31 @@
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
     // write code here
+    if (this.length === 0) {
+      return '';
+    }
+
+    let separateSign = separator;
```

Comment:
is this really necessary?

Reply:
i get this error if i do not create variable for parameter 'Assignment to function parameter 'separator''